tracer/build/_build/Build.GitHub.cs

```diff
@@ -94,6 +99,92 @@ await client.Issue.Update(
             Console.WriteLine($"PR assigned");
         });
 
+    Target LLMReport => _ => _
```

Comment:
Maybe we should rename it so that it's clear it's only a review of a PR? 🤔

Reply:
I have renamed the target to LLMPRReview. Thanks!